src/components/calltree/ProfileCallTreeView.js

```diff
@@ -5,9 +5,9 @@
 // @flow
 
 import React from 'react';
-import CallTree from './CallTree';
```

Comment:
For example that import statement is importing a file from the same directory. We should keep the `./` here instead of converting it.